File: php/testcode/benchmark_test_01248.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest01248(BenchmarkRequest $req): BenchmarkResponse {
    $field = $req->param('field');
    $value = $req->param('value');
    $store = [];
    $store[$field] = $value;
    return BenchmarkResponse::ok('set');
}

File: rust/testcode/benchmark_test_01248.rs
use regex::Regex;

pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let input = req.param("data");
    let pattern = Regex::new(r"^(a+)+$").unwrap();
    if pattern.is_match(&input) {
        super::shared::BenchmarkResponse::ok("Matched")
    } else {
        super::shared::BenchmarkResponse::ok("No match")
    }
}
